pkg/shell/base_index.js

```diff
@@ -356,8 +367,19 @@ var phantom_checkpoint = phantom_checkpoint || function () { };
             if (typeof data !== "string")
                 return;
 
-            var source = source_by_name[child.name];
-            var control;
+            var source, control;
+
+            /*
+             * On Internet Explorer we see Access Denied when non Cockpit
+             * frames send messages (such as Javascript console). This also
+             * happens when the window is closed.
+             */
+            try {
+                source = source_by_name[child.name];
+            } catch(ex) {
+                return;
```

Comment:
> How does such an ex for an Access Denied look like? Is there anything predictable in it which this catch could use to restrict this? 

This is really hard to do in a cross browser fashion. Javascript doesn't help us here at all, so we're limited to screen scraping, which is crazy in browser exceptions.

> Could this at least console.warn(ex) for debugging?

Sure. But it's not really a warning. It's us processing data from a window that no longer exists, we really should just ignore the window.